groups/elements.yml

```diff
@@ -0,0 +1,2 @@
+name: HTML elements
```

Comment:
How should the media elements group relate to this group? `<audio>` and `<video>` are elements, but the other things in the media elements group are `HTMLMediaElement` APIs.

Reply:
I would perhaps leave the media-elements group untouched, but make `<audio>` and `<video>` be both in elements and media-elements.
I guess my first intent here is, with the large number of new HTML elements being added, to have a nice group that contains them all.

Reply:
That makes sense to me!